src/status.rs

```diff
@@ -1,1601 +1,331 @@
-//! Status Codes
+//! HTTP status codes.
+
 use std::fmt;
 use std::num::{FromPrimitive, ToPrimitive};
-use std::mem::transmute;
-
-use std::cmp::Ordering::{self, Less, Equal, Greater};
+use std::cmp::Ordering;
 
-// shamelessly lifted from Teepee. I tried a few schemes, this really
-// does seem like the best.
+pub use self::StatusCode::*;
+pub use self::StatusClass::*;
```

Comment:
I'd rather keep these namespaced by enum for now. If there's much demand, this can be an immediate follow up, but it seems like a different decision than this PR.